### DISCUSSION TITLE
WHAT MEANS: Temporary Chat Activation: Integrated a new URL parameter 'temporary-chat=true' to enable temporary chat sessions directly through the URL.

### DISCUSSION THREAD

**Discussion by eggooanba** [open-webui/open-webui#4863] [General]
With the newest version you are promoting:
'Temporary Chat Activation: Integrated a new URL parameter 'temporary-chat=true' to enable temporary chat sessions directly through the URL.'

Could you please elaborate on this?

Thanks

**Comment by m8dhouse** [comment]
https://<yoururl/?q="tell me a fun fact"&models=gpt-4o-mini

it will open window, start new chat and create icon/title
You can also add a new parameter temporary-chat=true to have the chat temporary (so not added to history). 
No other options for the moment as far as I know.
